src/libs/checkForUpdates.ts

```diff
@@ -16,4 +13,4 @@ function checkForUpdates(platformSpecificUpdater: PlatformSpecificUpdater) {
     }, UPDATE_INTERVAL);
 }
 
-module.exports = checkForUpdates;
```

Comment:
This caused a regression https://github.com/Expensify/App/issues/36522. After changing the export syntax from CommonJS to ESM we had to check the files that depend on that module and update them accordingly.